src/website/app/demos/Pagination/examples/table.js

```diff
@@ -0,0 +1,63 @@
+/* @flow */
+import { Component } from 'react';
+import Pagination from '../../../../../library/Pagination';
+import Table from '../../../../../library/Table';
+import DemoLayout from '../../shared/DemoLayout';
+import { columns, data } from '../shared/data';
+
+export default {
+  id: 'table',
+  title: 'Table',
+  description: `Pagination can be used in conjunction with a
+[Table](/components/table) by providing sliced data calculated from current page
+and page size.`,
+  scope: { Component, columns, data, DemoLayout, Pagination, Table },
+  source: `
+    () => {
+      const pageSize = 3;
+
+      class PaginatedTable extends Component {
+        constructor(props) {
+          super(props);
+
+          this.state = {
+            currentPage: 1,
+          };
+
+          this.handlePageChange = this.handlePageChange.bind(this);
+        }
+
+        handlePageChange(currentPage) {
+          this.setState({ currentPage });
+        }
+
+        render () {
+          const { currentPage } = this.state;
+          const firstRow = (currentPage - 1) * pageSize;
+          const lastRow = (currentPage - 1) * pageSize + pageSize;
```

Comment:
@brentertz We thought that indexing currentPage at 1 would improve consumer experience because then it matches what's displayed; our internal code is also simpler. But then it requires the dev to calculate spliced data based on `currentPage - 1`; thoughts on whether we should change it to be 0-indexed?